Doc/c-api/structures.rst

```diff
@@ -306,9 +316,17 @@ There are these calling conventions:
    .. versionadded:: 3.7
 
 
-.. data:: METH_METHOD | METH_FASTCALL | METH_KEYWORDS
+.. c:macro:: METH_METHOD
+
+   Can only be used in the combination with other flags:
+   :ref:`METH_METHOD | METH_FASTCALL | METH_KEYWORDS <METH_METHOD-METH_FASTCALL-METH_KEYWORDS>`.
+
+
+.. _METH_METHOD-METH_FASTCALL-METH_KEYWORDS:
 
-   Extension of :const:`METH_FASTCALL | METH_KEYWORDS` supporting the *defining
+:c:expr:`METH_METHOD | METH_FASTCALL | METH_KEYWORDS`
+   Extension of :ref:`METH_FASTCALL | METH_KEYWORDS <METH_FASTCALL-METH_KEYWORDS>`
+   supporting the *defining
    class*, that is, the class that contains the method in question.
```

Comment:
Just a minor rewrap to avoid breaking `*defining class*` in half:
```suggestion
   supporting the *defining class*, that is,
   the class that contains the method in question.
```